iip_search/app.py

```diff
@@ -246,11 +248,8 @@ def get_inscription(
     #     if verification.get("status"):
     #         response.status_code = status.HTTP_404_NOT_FOUND
     #         return verification
-    try:
```

Comment:
Why is this try-except removed?

Reply:
You're right, it shouldn't be. Sorry, I must have missed it when merging my changes to app.py with yours.